builders/ethereum/dev-env/tenderly.md

```diff
@@ -77,28 +75,31 @@ After a contract has been added, it will appear in the list of contracts on the
 
-Tenderly makes creating a fork through the dashboard quite simple. To get started, click on **Forks** on the left-side menu and then click **Create Fork**. From there, you can take the following steps:
+Tenderly makes creating a TestNet through the dashboard quite simple. To get started, click on **Virtual TestNets** on the left-side menu and then click **Create Virtual TestNet**. From there, you can take the following steps:
 
-1. Select **Moonbeam** or **Moonriver** from the **Network** dropdown
-2. (Optional) Give your fork a name
-3. If you only need data up until a specific block, you can toggle the **Use Latest Block** slider to off and specify the block number. Otherwise, you can leave the slider as is to include all blocks up until the latest block
-4. Click **Create**
+1. Select **Moonbeam**, **Moonriver**, or **Moonbase Alpha** from the **Network** dropdown
+2. (Optional) Give your TestNet a name
+3. Select your **Chain ID** you can use a custom one or the original network ID. It is recommended to set a custom Chain ID to prevent replay attacks and avoid issues when adding the Virtual TestNet to wallets
+4. Choose whether to enable or disable the **Public Explorer**
+5. **State Sync** Enable this option if you want to keep your Virtual TestNet updated in real-time with the parent network
+6. If you only need data up until a specific block, you can toggle the **Use Latest Block** checkbox off and specify the block number. Otherwise, leave it enabled to include all blocks up until the latest block
+7. Click **Create**
 
-![Fork Moonbeam](/images/builders/ethereum/dev-env/tenderly/tenderly-5.webp)
+![Virtual TestNet Moonbeam](/images/builders/ethereum/dev-env/tenderly/tenderly-5.webp)
 
-Once you've created your fork, you can start using it by deploying a contract to it or creating a transaction simulation using it.
+Once you've created your Virtual TestNet, you can start using it by deploying a contract to it or creating a transaction simulation using it.
 
-To deploy a contract to your fork, you can click on the **Deploy Contract** button, upload your contract's source code, and set the compiler configurations. Once you submit the deployment, you'll see the transaction of your deployment appear under the **Simulated Transactions** tab and can click on the simulation for more information.
+To deploy a contract to your Virtual TestNet, click on  **Contracts**  in the left menu, You can use a contract from the **Watched Contracts** tab or upload your contract's address by clicking the **Watch Contract** button in the upper left corner. Once you submit the address, the contract will appear in the **Contracts** tab, where you can click on it to see more information.
```

Comment:
```suggestion
To deploy a contract, go to Contracts in the left menu. Use one from **Watched Contracts** or add a new one via **Watch Contract**. Once added, it will appear in **Contracts**, where you can view its details.
```

To make it more concise